src/appimagetool.c

```diff
@@ -53,7 +53,7 @@
 #include <limits.h>
 #include <stdbool.h>
 
-#include "appimage/appimage.h"
+#include <appimage/appimage_shared.h>
```

Comment:
Depending on appimage.h would introduce a dependency on the generated config.h. However it seems that including appimage_shared.h is enough to fix the build and make the tests pass. Same for validate.c.

Reply:
I agree. `validate.c` may be removed soon anyway, I recently published a replacement (which actually works even) built from AppImageUpdate's code base. https://github.com/AppImage/AppImageUpdate/releases